src/server/rdb_load.cc

```diff
@@ -433,9 +509,13 @@ void RdbLoaderBase::OpaqueObjLoader::operator()(const LzfString& lzfstr) {
 void RdbLoaderBase::OpaqueObjLoader::operator()(const unique_ptr<LoadTrace>& ptr) {
   switch (rdb_type_) {
     case RDB_TYPE_SET:
+    case RDB_TYPE_SET_INTSET:
```

Comment:
I am really not quite sure about this. 

The main question I have here, is when is this function called as opposed to  `void RdbLoaderBase::OpaqueObjLoader::operator()(const base::PODArray<char>& str)` 
and why the `RDB_TYPE_XXX` are handled differently via `HandleBlob`

Reply:
I now get it, different types are handled differently which depends on how the `OpaqueObj` is created